packages/core/src/components/forms/numericInput.tsx

```diff
@@ -524,6 +524,23 @@ export class NumericInput extends AbstractComponent<HTMLInputProps & INumericInp
     private isFloatingPointNumericCharacter(char: string) {
         return NumericInput.FLOATING_POINT_NUMBER_CHARACTER_REGEX.test(char);
     }
+
+    private getStepMaxPrecision(props: HTMLInputProps & INumericInputProps) {
+        let stepMaxPrecision = Utils.countDecimalPlaces(props.stepSize);
+        if (props.minorStepSize != null) {
+            stepMaxPrecision = Math.max(stepMaxPrecision, Utils.countDecimalPlaces(props.minorStepSize));
+        }
+
+        return stepMaxPrecision;
```

Comment:
Sweet, I like that better. 👍 This was a relic from a few commits ago, when I considered adding a configurable `precision` prop that we'd have to sanitize.